docs/user/integration/dynatrace/README.md

```diff
@@ -0,0 +1,154 @@
+# Integrate Kyma with Dynatrace
+
+## Overview 
+
+The Kyma Telemetry module supports you in integrating with observability backends in a convenient way. The following example outlines how to integrate with [Dynatrace](https://www.dynatrace.com) as a backend. With Dynatrace, you can get all your monitoring and tracing data into one observability backend to achieve real Application Performance Management with monitoring data in context.
+
+## Prerequisistes 
+
+- Kyma as the target deployment environment
+- Dynatrace account with permissions to create new access tokens
+- Helm 3.x if you want to deploy open-telemetry sample application
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace you want to use for Dynatrace as a variable. Replace the `{NAMESPACE}` placeholder in the following command and run it:
+
+    ```bash
+    export DYNATRACE_NS="{NAMESPACE}"
+    ```
+1. If you haven't created a Namespace yet, do it now:
+    ```bash
+    kubectl create namespace $DYNATRACE_NS
+    ```
+
+### Create access token
+
+Before connecting Kyma and Dynatrace, create a Dynatrace access token:
+
+1. In the Dynatrace navigation, go to the **Manage** > **Access tokens** > **Generate new token**.
+1. Type the name you want to give to this token. If needed, set an expiration date.
+1. Select the following scopes:
+   - **Ingest metrics**
```

Comment:
I assume **Ingest metrics** and **Ingest OTEL traces** is all you need, have you verified if the others are really needed, especially **Read metrics** sound strange